diff --git a/sbat.c b/sbat.c
index 446bed1a..0946dfbb 100644
--- a/sbat.c
+++ b/sbat.c
@@ -154,7 +154,8 @@ verify_single_entry(struct sbat_entry *entry, struct sbat_var *sbat_var_entry)
 {
 	UINT16 sbat_gen, sbat_var_gen;
 
-	if (strcmp(entry->component_name, sbat_var_entry->component_name) == 0) {
+	if (strcmp((const char *)entry->component_name,
+		   (const char *)sbat_var_entry->component_name) == 0) {
 		dprint(L"component %a has a matching SBAT variable entry, verifying\n",
 			entry->component_name);
 
@@ -162,8 +163,8 @@ verify_single_entry(struct sbat_entry *entry, struct sbat_var *sbat_var_entry)
 		 * atoi returns zero for failed conversion, so essentially
 		 * badly parsed component_generation will be treated as zero
 		 */
-		sbat_gen = atoi(entry->component_generation);
-		sbat_var_gen = atoi(sbat_var_entry->component_generation);
+		sbat_gen = atoi((const char *)entry->component_generation);
+		sbat_var_gen = atoi((const char *)sbat_var_entry->component_generation);
 
 		if (sbat_gen < sbat_var_gen) {
 			dprint(L"component %a, generation %d, was revoked by SBAT variable",
